cardano-node/docs/GenesisGeneration.md

```diff
@@ -0,0 +1,170 @@
+
+### blockVersionData
+
+
+The key `blockVersionData` contains fundamental blockchain-related values:
+
+- `heavyDelThd` - heavyweight delegation threshold. The delegate (i.e stakeholder) must possess no less than this threshold of stake in order to participate in heavyweight delegation.
+- `maxBlockSize` - maximum size of block in bytes.
+- `maxHeaderSize` - maximum size of block's header in bytes.
+- `maxProposalSize` - maximum size of Cardano SL update proposal in bytes.
+- `maxTxSize` - maximum size of transaction in bytes.
+- `mpcThd` - threshold for participation in SSC (shared seed computation). See SSC [here.](https://cardanodocs.com/glossary/) This will be deprecated in the OBFT era.
+- `scriptVersion` - Redundant. This was meant to be the version of Plutus however there are no smart contracts in the Byron era.
+- `slotDuration` - slot duration, in milliseconds.
+- `softforkRule` - Values defining softfork resolution rule. When the stake belonging to block issuers, issuing a given block version, is greater than the current softfork resolution threshold, this block version is adopted. The current softfork resolution threshold is determined as follows: If a proposal is confirmed at the `c`-th epoch, softfork resolution threshold at a later `t`-th epoch will be `max minThd (spInitThd - (t - c) * spThdDecrement)`.
+
+    - `initThd` - Initial threshold (right after proposal is confirmed).
+    - `minThd` - minimal threshold (i.e. threshold can't become less than this one).
+    - `thdDecrement` - theshold will be decreased by this value after each epoch.
+- `txFeePolicy` -  Transaction fee policy represents a formula to compute the minimal allowed Fee for a transaction. Transactions with lesser fees won't be accepted. At the moment we have just one policy type (a linear Equation on the transaction size, fee = summand + multiplier * txsize), but in the future other policies may Be added.
+    - `summand`
+    - `multipler`
+- `unlockStakeEpoch` - This has been re-purposed for unlocking the OuroborosBFT logic in the software. Relevant: [CDEC-610](https://iohk.myjetbrains.com/youtrack/issue/CDEC-610)
+- `updateImplicit` - Time to live for a protocol update proposal.
+  The number of slots a proposal has to gather a majority of votes. If a
+  majority of votes has not been reached before this period, then the
+  proposal is rejected.
+- `updateProposalThd` - threshold for Cardano SL update proposal.
+- `updateVoteThd` - threshold for voting for Cardano SL update proposal.
+
+Please note that values of all thresholds are measured in Lovelaces.
+
+1 Lovelace = 1/100,000 ADA
+
+### bootStakeholders
+The key `bootStakeholders` contains bootstrap era stakeholders' identifiers (StakeholderIds) with corresponding weights.
+
+### ftsSeed
+
+The key `ftsSeed` contains seed value required for Follow-the-Satoshi mechanism (hex-encoded).
```

Comment:
@Jimbo4350, I couldn't find any references to its use in the new codebase.  Perhaps it's been deprecated as well?

Reply:
Yes, I believe this will also be deprecated as the output of the `SSC` is used in the Follow-The-Satoshi protocol. I will document this.